x-pack/plugins/search_indices/public/components/shared/create_index_panel.tsx

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import {
+  EuiButtonEmpty,
+  EuiButtonGroup,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTextAlign,
+  EuiTitle,
+  useEuiTheme,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { docLinks } from '../../../common/doc_links';
+import { useKibana } from '../../hooks/use_kibana';
+import { CreateIndexViewMode } from '../../types';
+
+const MAX_WIDTH = '650px';
+
+export interface CreateIndexPanelProps {
+  children: React.ReactNode | React.ReactNode[];
+  createIndexView: CreateIndexViewMode;
+  onChangeView: (id: string) => void;
+  onClose: () => void;
+  showSkip?: boolean;
+  title?: React.ReactNode;
+}
+
+export const CreateIndexPanel = ({
+  children,
+  createIndexView,
+  onChangeView,
+  onClose,
+  showSkip,
+  title,
+}: CreateIndexPanelProps) => {
+  const { cloud, http } = useKibana().services;
+  const { euiTheme } = useEuiTheme();
+
+  const o11yTrialLink = useMemo(() => {
+    if (cloud && cloud.isServerlessEnabled) {
+      const baseUrl = cloud?.projectsUrl ?? 'https://cloud.elastic.co/projects/';
+      return `${baseUrl}create/observability/start`;
+    }
+    return http.basePath.prepend('/app/observability/onboarding');
+  }, [cloud, http]);
+
+  return (
+    <>
+      <EuiPanel
+        color="subdued"
+        hasShadow={false}
+        hasBorder
+        style={{
+          maxWidth: MAX_WIDTH,
+          margin: '0 auto',
+          padding: euiTheme.size.l,
+          paddingTop: euiTheme.size.m,
+        }}
+      >
+        <EuiFlexGroup justifyContent="flexEnd">
+          <EuiButtonIcon
+            data-test-subj="closeCreateIndex"
+            iconType="cross"
+            onClick={onClose}
+            color="text"
+            aria-label={i18n.translate('xpack.searchIndices.createIndex.closeAriaLabel', {
+              defaultMessage: 'Close create index',
+            })}
+          />
+        </EuiFlexGroup>
+        <EuiPanel
+          color="transparent"
+          paddingSize="none"
+          style={{
+            paddingLeft: euiTheme.size.m,
+            paddingRight: euiTheme.size.m,
+          }}
+        >
+          <EuiFlexGroup alignItems="center" gutterSize="m">
+            <EuiFlexItem grow={false}>
+              <EuiIcon type="logoElasticsearch" size="xl" />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiTitle size="xs">
+                <h1>
+                  {i18n.translate('xpack.searchIndices.startPage.pageTitle', {
```

Comment:
nitpick: should these localisation ids be `xpack.searchIndices.createIndex.pageTitle` ?